exec/java-exec/src/test/java/org/apache/drill/exec/store/avro/AvroFormatTest.java

```diff
@@ -170,25 +169,35 @@ public void testSimplePrimitiveSchema_SelectColumnSubset() throws Exception {
 
   @Test
   public void testSimplePrimitiveSchema_NoColumnsExistInTheSchema() throws Exception {
-    final String file = generateSimplePrimitiveSchema_NoNullValues().getFileName();
-    try {
-      test("select h_dummy1, e_dummy2 from dfs.`%s`", file);
-      Assert.fail("Test should fail as h_dummy1 and e_dummy2 does not exist.");
-    } catch(UserException ue) {
-      Assert.assertTrue("Test should fail as h_dummy1 and e_dummy2 does not exist.",
-          ue.getMessage().contains("Column 'h_dummy1' not found in any table"));
-    }
+    final String file = generateSimplePrimitiveSchema_NoNullValues(1).getFileName();
+    testBuilder()
+      .sqlQuery("select h_dummy1, e_dummy2 from dfs.`%s`", file)
+      .unOrdered()
+      .baselineColumns("h_dummy1", "e_dummy2")
+      .baselineValues(null, null)
+      .go();
   }
 
   @Test
   public void testSimplePrimitiveSchema_OneExistAndOneDoesNotExistInTheSchema() throws Exception {
-    final String file = generateSimplePrimitiveSchema_NoNullValues().getFileName();
-    try {
-      test("select h_boolean, e_dummy2 from dfs.`%s`", file);
-      Assert.fail("Test should fail as e_dummy2 does not exist.");
-    } catch(UserException ue) {
-      Assert.assertTrue("Test should fail as e_dummy2 does not exist.", true);
-    }
+    final String file = generateSimplePrimitiveSchema_NoNullValues(1).getFileName();
+    testBuilder()
+      .sqlQuery("select h_boolean, e_dummy2 from dfs.`%s`", file)
+      .unOrdered()
+      .baselineColumns("h_boolean", "e_dummy2")
+      .baselineValues(true, null)
+      .go();
+  }
+
+  @Test
+  public void testImplicitColumnFilename() throws Exception {
+    final String file = generateSimplePrimitiveSchema_NoNullValues(1).getFileName();
+    testBuilder()
+      .sqlQuery("select filename from dfs.`%s`", file)
```

Comment:
Please test all implicit columns and at least one partition column.

Reply:
Thanks for pointing this, modified existing test to check except the `filename` also `suffix`, `fqn` and `filepath` implicit columns. Added separate test for partition column.